src/AbstractMenu.js

```diff
@@ -63,6 +67,11 @@ export default class AbstractMenu extends Component {
         const { selectedItem } = this.state;
         const children = [];
         const childCollector = (child) => {
+            // child can be empty in case you do conditional rendering of components, in which case it should not be accounted for as a real child
+            if (child === null || child === false || child === undefined || child === '') {
```

Comment:
Yeah I just started with null/false but then it grew out of control ;) I guess it's fine.